src/main/webapp/mentors.js

```diff
@@ -2,10 +2,10 @@
  * Get mentors from the mentor servlet.
  */
 function getMentors() {
-  console.log("entering get mentors function/n");
+  console.log('entering get mentors function/n');
   let url = new URL('/mentors', location.protocol + '//' + location.hostname);
 
-  fetch(url).then(errorHandling(response)).then(response => response.json()).then(mentors => {
+  fetch(url).then(errorHandling).then(response => response.json()).then(mentors => {
```

Comment:
This looks good! However, you need to add a catch block at the end of this to catch the errors you're throwing and the errors that can be thrown from the fetch call.

Reply:
Thank you! I will add the catch block.